utils/sl/prereq_tools/base.py

```diff
@@ -811,6 +811,11 @@ def setup_parallel_build(self):
         """Set the JOBS_OPT variable for builds"""
         jobs_opt = GetOption('num_jobs')
         self.__env["JOBS_OPT"] = "-j %d" % jobs_opt
+        #Multiple go jobs can be running at once via the -j option so limit each
+        #to 1 proc.   This allows for compilation to continue on systems with
+        #limited processor resources where the number of go procs will be
+        #multiplied by jobs_opt.
+        self.__env["ENV"]["GOMAXPROCS"] = "1"
```

Comment:
I guess this is fine, but it seems like it would be better to detect when it's necessary rather than hamstringing builds in all environments.

Reply:
problem is I don't really know.   I originally set this to num_jobs and then set ulimit -u 200.  With that setting, I couldn't run with -j24.  I suppose we could check ulimit and set this for anything less than 4096 or some such to be safe

Reply:
I should add that I didn't notice a speed difference after adding this

Reply:
OK, I'm not overly concerned about it and if it fixes the problem on Frontera then I'm fine with it.